packages/glimmer-runtime/tests/initial-render-test.ts

```diff
@@ -806,105 +785,8 @@ test("Case-sensitive tag has capitalization preserved", function() {
   compilesTo('<svg><linearGradient id="gradient"></linearGradient></svg>');
 });
 
-// QUnit.skip("svg can live with hydration", function() {
-//   let template = compile('<svg></svg>{{name}}');
-
-//   let fragment = render(template, { name: 'Milly' }, env, { contextualElement: document.body }).fragment;
-
-//   equal(
-//     fragment.childNodes[0].namespaceURI, svgNamespace,
-//     "svg namespace inside a block is present" );
-// });
-
-// QUnit.skip("top-level unsafe morph uses the correct namespace", function() {
-//   let template = compile('<svg></svg>{{{foo}}}');
-//   let fragment = render(template, { foo: '<span>FOO</span>' }).fragment;
-
-//   equal(getTextContent(fragment), 'FOO', 'element from unsafe morph is displayed');
-//   equal(fragment.childNodes[1].namespaceURI, xhtmlNamespace, 'element from unsafe morph has correct namespace');
-// });
-
-// QUnit.skip("nested unsafe morph uses the correct namespace", function() {
-//   let template = compile('<svg>{{{foo}}}</svg><div></div>');
-//   let fragment = render(template, { foo: '<path></path>' }).fragment;
-
-//   equal(fragment.childNodes[0].childNodes[0].namespaceURI, svgNamespace,
-//         'element from unsafe morph has correct namespace');
-// });
-
-// QUnit.skip("svg can take some hydration", function() {
-//   let template = compile('<div><svg>{{name}}</svg></div>');
-
-//   let fragment = render(template, { name: 'Milly' }).fragment;
-//   equal(
-//     fragment.firstChild.childNodes[0].namespaceURI, svgNamespace,
-//     "svg namespace inside a block is present" );
-//   equalTokens( fragment.firstChild, '<div><svg>Milly</svg></div>',
-//              "html is valid" );
-// });
-
-// QUnit.skip("root svg can take some hydration", function() {
```

Comment:
Moved to `updating-test.ts` as INUR, renamed `expression nested inside a namespaced root element`